packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -405,9 +405,15 @@ export const Checkout = ({
 
           <PaymentMethodHeader
             plan={selectedPlan}
-            onClick={() => setCheckboxSet(!checkboxSet)}
+            onClick={() => {
+              const checkboxValue = !checkboxSet;
+              setCheckboxSet(checkboxValue);
+              const errorValue = !checkboxValue;
+              setCheckboxRequiredError(errorValue);
```

Comment:
I'm slightly confused. This seems to be creating two variables which looks like it's inverting `checkboxSet` into a local `checkboxValue` variable and then inverting it back and assigning it back to a local `errorValue` variable.

Wouldn't the code just be something like this:
```js
setCheckboxSet(!checkboxSet);
setCheckboxRequiredError(checkboxSet);
```